### PR TITLE
Set up Jest snapshot testing

### LINKED ISSUE #6: Add a basic layout with example component test
- Bootstrap v4 alpha for the framework
- Top-level component with a basic layout
- Setup Jest snapshot tests

### PR DESCRIPTION
**Pull request by conzett**
More layout work needed.
Closes #6

### CODE REVIEW

**Comment by daveshah** [comment]
I finally got around to reading the snapshot testing docs in Jest. Glad you added this - I dig it. LGTM. 👍

**Review by daveshah — DISMISSED** [review]

**Comment by conzett** [comment]
I really like the snapshot approach, it's pretty much the same idea as the response fixtures in Criteria Builder. The assertion doesn't need to parse or check every detail of the HTML just make someone acknowledge and review the change before passing.

This should also give us a first test to use for #3 🤣

**Comment by brendanthomas1** [comment]
This good to go? I was going to try and knock out a component this weekend and would like to add tests.

**Comment by conzett** [comment]
@brendanthomas1 I think it's pretty close - I was going to do some final cleanup to remove some of the commented out stuff, old assets and "Welcome to React" parts, but I think it's mostly done. Remind me this afternoon and I can get that cleaned up real quick 😁

**Comment by conzett** [comment]
@daveshah, @brendanthomas1 "pet store" example is integrated into both the example index page and the specs ( ✅  Passing). Let me know what you think!

File: src/App.js [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
+            <br/>
+            <div className="media">
+              <img
+                style={{ width: '64px', height: '64px', background: "gray" }}
+                className="d-flex mr-3"
+                alt="React logo"
+                src={logo}/>
+              <div className="media-body">
+                <h1 className="h5">Megalodoc</h1>
+                <em>That's a bad fish</em>
```
Comment (austenmadden) [inline src/App.js]: Just out of curiosity is there a way to escape the ' and make the git diff happy or is that just unavoidable. Super minor, ignore me 
disappear

File: src/App.js [hunk 2]
```diff
@@ -1,21 +1,33 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
```
Comment (brendanthomas1) [inline src/App.js:1]: Should this be `.jsx`?

**Review by brendanthomas1 — APPROVED** [review]
I really don't have much to add to this. I think it's a good jumping-off point 👍

File: src/App.js (hunk 2 above)
Comment (conzett) [reply src/App.js:1]: I think since Babel does JSX now this just gets transformed as well. I could be wrong about that though.

**Review by daveshah — APPROVED** [review]
Looks good 👍

**Pull request merged**